vunit/ui/results.py

```diff
@@ -24,5 +29,19 @@ def merge_coverage(self, file_name, args=None):
         :param file_name: The resulting coverage file name.
         :param args: The tool arguments for the merge command. Should be a list of strings.
         """
-
         self._simulator_if.merge_coverage(file_name=file_name, args=args)
+
+    def get_report(self):
+        """
+        Get a report (dictionary) of tests: status, time and output path
+        """
+        report = {}
+        for (
+            test
+        ) in self._report._test_results_in_order():  # pylint: disable=protected-access
+            name, obj = test.to_dict()
+            obj["path"] = re.sub(
+                "^%s" % join(self.output_path, "test_output") + sep, "", obj["path"]
```

Comment:
Duplication of information with hard-coded "test_output"

Reply:
Also at least wrap the regex operation in a function that documents what it wants to achive.

Reply:
> Duplication of information with hard-coded "test_output"

I don't understand what you mean.

Reply:
There is somewhere else in the code where the test output path was decided. If that path changed the code in the PR would be wrong since it duplicates the knowledge of where the test output path is.

Reply:
Now, `TEST_OUTPUT_PATH = "test_output"` is defined in `vunit.ui.common` and it is imported by both `vunit.ui.vunit` and `vunit.ui.results`.